test/e2e/bench/bench_test.go

```diff
@@ -60,7 +61,7 @@ func TestBench(t *testing.T) {
 var _ = BeforeSuite(func() {
 	var found bool
 	reportDir, found = os.LookupEnv("CONTOUR_BENCH_REPORT_DIR")
-	require.True(f.T(), found, "Must provide CONTOUR_BENCH_REPORT_DIR env var")
+	assert.True(f.T(), found, "Must provide CONTOUR_BENCH_REPORT_DIR env var")
```

Comment:
Did we need to switch all these `test/e2e` uses of `require`  over then? The linter passes even if I switch 'em back locally. I guess not a crisis either way, the test will still fail eventually regardless, but failing fast where necessary with `require` is nice for debuggability

Reply:
just tried it out and ginkgo/gomega's failure handler will abort a test immediately with either assert or require, I guess since assert calls Fail vs. require calling FailNow

```go
package ginkgofail_test

import (
	"fmt"
	"testing"

	. "github.com/onsi/ginkgo/v2"
	. "github.com/onsi/gomega"
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
)

func TestHTTPProxy(t *testing.T) {
	RegisterFailHandler(Fail)
	RunSpecs(t, "foo")
}

var _ = Describe("bar", func() {
	It("assert", func() {
		assert.True(GinkgoT(), false)

		fmt.Fprintln(GinkgoWriter, "shouldn't see this")
	})

	It("require", func() {
		require.True(GinkgoT(), false)

		fmt.Fprintln(GinkgoWriter, "shouldn't see this either")
	})
})
```

Reply:
I can switch the test/e2e's back so we have less of a diff though

Reply:
Oh, good to know. In that case I have no issue as-is